testing/upload_validation.ts

```diff
@@ -873,6 +873,7 @@ const genericSyncTableSchema = zodCompleteObject<GenericSyncTableDef & {isDynami
   // it in the validator like this helps zod flag it in the way we need.
   isDynamic: zodDiscriminant(false).optional(),
   getSchema: formulaMetadataSchema.optional(),
+  identityName: z.string().optional(),
```

Comment:
How come it's optional here?

Reply:
oh do we intend to make this a breaking change? probably a good time to do this I guess?

Reply:
oh never mind. saw the other comment.

Reply:
Oops, forgot to update that, good catch.